### ISSUE TITLE
Snapshot tests returns {{value}} rather than the interpolated value.

### ISSUE THREAD

**Issue by dinosaurjoe** [i18next/react-i18next#524]
Hello,

I am attempting to create translations for the project that I am working on and am doing testing via jest. When the site is live, the interpolation works, however when running snapshot tests it returns the value of the key with the interpolation prefix/suffix.

For example, the expected translated, interpolated string should be `Please contact your local Demo Shop office for more info.`

However, I am receiving `Please contact your local {{tenant}} office for more info.` on my actual snapshot.

All of the other keys are translating correctly. Is there some specific set up that needs to be done in order to test for interpolated strings?

**Comment by dinosaurjoe** [comment]
I think may be misunderstanding something. So far with my translations I haven't had to do any mocking in order to see the proper translations within my snapshots. I am importing my translated components to my jest file, it is just when I interpolate on a translation that I am unable to see the translation.

**Comment by jamuhl** [comment]
nope...there is no reason...as long as t lookup find it's key - get the value - has the values to interpolate it will work -> that's why i guess your setup generally not working correctly.

you say you import your components to JEST -> i guess that is a component wrapped by the translate hoc -> so without mocking or having a i18next instance passed down like: https://github.com/i18next/react-i18next/blob/master/test/trans.render.spec.js#L31 it does not work...so i guess you mock i18n away like in the example repo....but idk...not having your code at hand....

**Comment by dinosaurjoe** [comment]
Here is an example of how I am working on this. I simplified most of the component/tests to see if that helps clarify the issue a bit.

Here is my component:
`import React, { Component } from 'react'
  import { translate } from 'react-i18next'

   class CargoDetails extends Component {
       render() {
         const { t, tenant } = this.props

         return (
             <p>
                  {t('cargo:insuranceContact', { tenant: tenant.name})}
             </p>
         )
      } 
     }

export default translate( 'cargo')(CargoDetails)
`

Here is my .json:

`"cargo"{"insuranceContact": "Please contact your local {{tenant}} office for more info."}"`

And here is the test that is failing:

`import * as React from 'react'
  import { shallow } from 'enzyme'
  import CargoDetails from './CargoDetails'
  
  const props = {
     tenant: { 
          name: 'Demo Shop'
     }
    }

  test('expect shallow to match', () => {
       expect(shallow(<CargoDetails {...props} />)).toMatchSnapshot()
   })
})`

**Comment by dinosaurjoe** [comment]
Same, I'm picking up Jest/Enzyme myself. I will mess around with a mount render and will update this if I find a solution to the problem. Thank you for your help and for building this project.

**Comment by jamuhl** [comment]
Would be great if you share when you see any improvement we could add to the test project (https://github.com/i18next/react-i18next/tree/master/example/test-jest) or the docs (https://react.i18next.com/misc/testing)

**Comment by dinosaurjoe** [comment]
will do @jamuhl
